decoration.sk

```diff
@@ -778,3 +778,49 @@ the wild update:
 
 	# sculk stuffs
 	{waterloggable} {surface} sculk vein¦s = minecraft:sculk_vein
+
+trails and tales update:
+	minecraft version = 1.20 or newer
+
+	[any] sapling¦s = any sapling, cherry sapling
+	[any] lea(f|ves) = any leaves, cherry leaves
+
+	{flower amount}:
+		{default} = -
+		with (one|1) flower = -[flower_amount=1]
+		with (two|2) flowers = -[flower_amount=2]
+		with (three|3) flowers = -[flower_amount=3]
+		with (four|all|4) flowers = -[flower_amount=4]
+	pink petals¦s {flower amount} = minecraft:pink_petals
+
+	torch flower¦s = minecraft:torchflower
+	{block half} pitcher plant¦s = minecraft:pitcher_plant
+
+	{hanging sign attachment}:
+		{default} = -
+		(not attached|straight) = -[attached=false]
+		(dangling|attached|loose) = -[attached=true]
+	{waterloggable} {directional} {wood type} wall hanging sign¦s = -wall_hanging_sign
+	{waterloggable} {rotatable} {hanging sign attachment} {wood type} hanging sign¦s = -hanging_sign
+	[any] oak sign¦s = oak sign, oak hanging sign, oak wall hanging sign
+	[any] birch sign¦s = birch sign, birch hanging sign, birch wall hanging sign
+	[any] jungle sign¦s = jungle sign, jungle hanging sign, jungle wall hanging sign
+	[any] spruce sign¦s = spruce sign, spruce hanging sign, spruce wall hanging sign
+	[any] acacia sign¦s = acacia sign, acacia hanging sign, acacia wall hanging sign
+	[any] dark oak sign¦s = dark oak sign, dark oak hanging sign, dark oak wall hanging sign
+	[any] mangrove sign¦s = mangrove sign, mangrove hanging sign, mangrove wall hanging sign
```

Comment:
Including hanging signs in the `... sign` pattern will be a breaking change for users who expect the item to be a regular sign.
Hanging signs are a separate item with a separate ID, it is not like a regular sign item being placed on the wall/floor.

Reply:
https://discord.com/channels/135877399391764480/828337517471203328/1156265882381713489

Reply:
I agree with Kenzie here - hanging signs should not be included here. This is a case where the any keyword should be required.